src/components/molecules/HomeTextBox.vue

```diff
@@ -1,19 +1,19 @@
 <template>
 	<div class="home__text-box">
-		<homeTextWHite v-bind:textWhite="text.textWhite"/>
-		<homeTextWHite v-bind:textCrimson="text.textCrimson"/>
+		<HomeTextWhite v-bind:textWhite="text.whiteText"/>
+		<HomeTextCrimson v-bind:textCrimson="text.crimsionText"/>
```

Comment:
why text crimson, you pass a text, dont define content of prop by propname, othervise component will be used only here